ui/src/views/user/signup/SignupView.vue

```diff
@@ -1,20 +1,171 @@
 <template>
   <v-container
-    id="signup-view"
-    class="fill-height text-center"
+    class="signup-view fill-height"
     tag="section"
   >
     <v-row justify="center">
-      <v-col cols="auto">
-        signup view
+      <v-col :cols="isMobile? 12 : 8">
+        <v-card
+          class="signup-view__card"
+          :elevation="isMobile? 0 : 2"
+        >
+          <v-card-title class="signup-view__title">
+            Sign Up
+          </v-card-title>
+          <v-card-text class="signup-view__description">
+            Have an account already? <router-link to="/user/login">
+              Log in here
+            </router-link>
+          </v-card-text>
+          <ValidationObserver v-slot="{ handleSubmit }">
+            <v-form
+              v-model="isFormValid"
+              class="signup-form"
+              @keyup.native.enter="handleSubmit(onFormSubmit)"
+              @submit.prevent="handleSubmit(onFormSubmit)"
+            >
+              <v-container>
+                <v-row>
+                  <v-col cols="12">
+                    <ValidationProvider
+                      v-slot="{ errors }"
+                      name="E-mail"
+                      rules="required|email"
+                    >
+                      <v-text-field
+                        v-model="formData.email"
+                        :rules="[() => !!formData.email || 'This field is required']"
+                        placeholder="email@email.com"
+                        outlined
+                        required
+                        :error-messages="errors[0]"
+                        label="E-mail"
+                      />
+                    </ValidationProvider>
+                  </v-col>
+
+                  <v-col cols="12">
+                    <ValidationProvider
+                      v-slot="{ errors }"
+                      name="Password"
+                      rules="required|alpha|min:8"
```

Comment:
Please refer to the newest PR, rebase the branch and remove the `alpha` rule. I added this rule previously because I forgot that password may contain symbols XD